.github/workflows/pr-to-update-chart.yml

```diff
@@ -0,0 +1,65 @@
+name: PR to update Charts
+
+# This action will run after a tag starting with "v" is published
+on:
+  registry_package:
+    types: [published]
+  workflow_dispatch:
+
+env:
+  LATEST_HEADLAMP_TAG: latest
+jobs:
+  create_pr_to_upgrade_chart:
+    name: Create PR to update Headlamp's version in the Helm Chart
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout headlamp repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Configure Git
+        run: |
+          user=${{github.actor}}
+          if [ -z $user ]; then
+            user=joaquimrocha
+          fi
+          git config --global user.name "$user"
+          git config --global user.email "$user@users.noreply.github.com"
+      - name: Get headlamp latest tag
+        run: |
+          latestTag=$(git tag --list --sort=version:refname 'v*' | tail -1)
+          echo "LATEST_HEADLAMP_TAG=$latestTag" >> $GITHUB_ENV
+          echo $latestTag
+      - name: Update Headlamp version in charts
+        run: |
+          if [ -z $user ]; then
+            user=joaquimrocha
+          fi
+          # Extract current version
+          CURRENT_CHART_VERSION=$(grep '^version:' ./charts/headlamp/Chart.yaml | cut -d' ' -f2)
+          CURRENT_CHART_VERSION=$(echo $CURRENT_CHART_VERSION | awk -F. -v OFS=. '{$(NF - 1) += 1 ; print}')
+          # Create branch
+          BRANCH=update_chart_headlamp_$HEADLAMP_VERSION
```

Comment:
`$HEADLAMP_VERSION` is not set anywhere, perhaps you wanted this to be `$LATEST_HEADLAMP_TAG`?